task/generate-olm-bundle/0.1/generate-olm-bundle.yaml

```diff
@@ -0,0 +1,40 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: generate-olm-bundle
+spec:
+  description: Generate an OLM bundle
+  params:
+    - name: input-dir
+      description: Directory to read cluster-ready operator manifests from
+      default: deploy
+    - name: channels
+      description: Comma-separated list of channels the bundle belongs to
+      default: alpha
+    - name: version
+      description: Semantic version of the operator in the generated bundle
+    - name: package-name
+      description: Bundle's package name
+  workspaces:
+    - name: source
+      description: Workspace with the source code
+  steps:
+    - name: generate-olm-bundle
+      # TODO: ensure this image is built within konflux
+      # TODO: can this be configurable as a parameter? it should be specific to
+      # the target version
+      image: "registry.redhat.io/openshift4/ose-operator-sdk-rhel9:v4.16"
```

Comment:
We should not use pullspecs for registry.redhat.io as these definitions are available to the community to use. They might not have Red Hat entitlements.

Reply:
I'm afraid I don't understand. `fbc-validation` uses `registry.redhat.io/openshift4/ose-operator-registry-rhel9`

Reply:
@arewm in this case, any suggestions on non registry.redhat.io images that fulfill the same use-case?  We were recommended to use this image by the OLM team if I recall!

Reply:
We do not have a better solution for this today, so I am going to resolve this thread for now. If Konflux users do not have Red Hat entitlements then they wouldn't be able to use a couple of the tasks.